pallene/translator.lua

```diff
@@ -0,0 +1,81 @@
+local translator = {}
+
+local function translate_expression(expression)
+    if expression._tag == "ast.Exp.Integer" then
+        io.write(expression.value)
+    end
+end
+
+local function translate_tl_var(node)
+    io.write('local ')
```

Comment:
When we print the `"local "`, the `declaration.name` and the and `" = "` I think we won't be preserving the the whitespace that might have been present around those tokens in the original input. Would it be possible to implement these parts similarly to what we have been doing with the type annotations? Just find the location of the `local` or `export` token and replace just that substring.